gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsBatchTest.java

```diff
@@ -476,6 +556,10 @@ public void testGetChangeRequest() {
 
   @Test
   public void testGetChangeRequestWithOptions() {
+    EasyMock.verify(dns);
+    EasyMock.reset(dns);
+    EasyMock.expect(dns.options()).andReturn(optionsMock);
+    EasyMock.replay(dns);
```

Comment:
Also here shouldn't we check that the callback correctly forwards the result/exception?